pkg/network/config/config_linux_test.go

```diff
@@ -33,3 +38,42 @@ func TestDisableRootNetNamespace(t *testing.T) {
 	defer ns.Close()
 	require.True(t, ns.Equal(rootNs))
 }
+
+func TestEventStreamEnabledForSupportedKernelsLinux(t *testing.T) {
+	t.Run("for kernels <4.15.0", func(t *testing.T) {
+		if runtime.GOOS != "linux" {
+			t.Skip("This is only for linux")
+		}
```

Comment:
nit: I think you can leave this out. The filename ends with `linux` and that means the tests will only run on linux so you can assume that all these tests are linux only (the other tests in this file do the same)